hack/dockerfiles/generated-files.Dockerfile

```diff
@@ -0,0 +1,53 @@
+# Forked from https://github.com/moby/buildkit/blob/v0.10.4/hack/dockerfiles/generated-files.Dockerfile
```

Comment:
This has changed a bit, see https://github.com/moby/buildkit/pull/3236

I think we can do the same here.

Reply:
Thanks for the suggestion. Updated this file.